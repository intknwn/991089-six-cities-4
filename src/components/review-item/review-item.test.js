import React from 'react';
import renderer from 'react-test-renderer';
import ReviewItem from './review-item.jsx';

const review = {
  "id": 3,
  "user": {
    "id": 18,
    "is_pro": true,
    "name": `Sophie`,
    "avatar_url": `https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/9.jpg`
  },
  "rating": 3,
  "comment": `Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.`,
  "date": `2020-07-17T16:06:01.820Z`
};

it(`ReviewItem render`, () => {

  const tree = renderer.create(
      <ReviewItem
        review={review}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
