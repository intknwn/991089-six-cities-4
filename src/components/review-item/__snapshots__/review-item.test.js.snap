// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem render 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Sophie"
        className="reviews__avatar user__avatar"
        height={54}
        src="https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/9.jpg"
        width={54}
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Sophie
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.
    </p>
    <time
      className="reviews__time"
      dateTime="2020-07-17T16:06:01.820Z"
    >
      July 2020
    </time>
  </div>
</li>
`;
